### ISSUE TITLE
Safari Web Extension Support

### ISSUE THREAD

**Issue by dwhiffing** [Tencent/MMKV#887]
### The language of MMKV
Swift

### The version of MMKV
any version

### The platform of MMKV
iOS

### The installation of MMKV
git clone

### What's the issue?
Looking to use MMKV to share information between my Safari Web Extension and it's native iOS app.  I'm new to App Extensions and Safari Web Extensions, but followed the install instructions as though I were building an App Extension.

I found I am able to build until I try to import the MMKV/MMKV.h file in the bridging header for my Web Extension.

Could you clarify if this should be possible with MMKV before I provide more information?

Thank you for your time.

**Comment by lingol** [comment]
You should use the MMKV app extension podspec for your app extension. The wiki has it all. You should check it more carefully.

**Comment by lingol** [comment]
https://github.com/Tencent/MMKV/wiki/iOS_setup

**Comment by dwhiffing** [comment]
Thank you, I did read those directions a number of times, and initially tried using cocoapods.  But ran into this error when trying to build the app when I try to use MMKV inside my SafariWebExtensionHandler.swift file:

`Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_MMKV"`

Based on my limited understanding, this means that while my bridging header linked <MMKVAppExtension/MMKV.h> correctly, it is not being compiled properly during build and therefore the required symbols are not defined.

I tried adding the `-ObjC` linker flag and including `libz` in the `Linked Binaries and Libraries` section as a shot in the dark.

I understand this isn't a place for technical support, but would be happy to contribute a working SafariWebExtension example as a PR if you could help me resolve this.

**Comment by dwhiffing** [comment]
Interesting.  After a good amount of reading, I found a suggestion to add `$(inherited)` to Other Linker Flags.  I added this to my Safari Web Extension target and the app is now building.  

I'm hoping this is the end of my challenges.  If I am successful and you are interested, I will include a functioning repo with a safari web extension which shares data between the app and web extension.

Thank you for providing this as open source.
